docs/source/overview.rst

```diff
@@ -248,8 +248,8 @@ Let's say we want to give each  point a name, we can create a feature like this:
 
 .. code-block:: python
 
-		spot_names = block.create_data_array('spot ids', 'nix.feature', data=['a', 'b'])
 		spot_names.append_set_dimension()
+		spot_names = block.create_data_array('spot ids', 'nix.feature',dtype=nix.DataType.Int8, data=[1, 2])
```

Comment:
this would not work: you cannot append a dimension before the DataArray exists. Please invert the order of lines 251 and 252.

Could you also change the introductory sentence to: "Let's say we want to give each point an ID, we can create a feature like this:"